addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/PathTraversalScanRule.java

```diff
@@ -343,8 +362,8 @@ public void scan(HttpMessage msg, String param, String value) {
             }
 
             // Check 2: Start detection for *NIX patterns
-            // note that depending on the AttackLevel, the number of prefixes that we will try
-            // changes.
+            // note that depending on the AttackLevel, additional variants including a null byte
+            // will be tried.
```

Comment:
I think this comment could just be removed, it seems clear just looking at the code.

Reply:
fixed